src/reducers/versions.spec.tsx

```diff
@@ -1774,4 +1778,203 @@ describe(__filename, () => {
       expect(state.versionInfo[beginFetchVersionId]).toEqual(undefined);
     });
   });
+
+  type TestHunkChange = {
+    lineNumber: number;
+    type: string;
+  };
+
+  type TestHunk = TestHunkChange[];
+
+  // This helper accepts an array of arrays, each of which is a list of changes
+  // which will be contained in a hunk.
+  const createDiff = (testHunks: TestHunk[]): DiffInfo => {
+    const diffSample = parseDiff(diffWithDeletions)[0];
```

Comment:
It would help to put a comment above this saying how it's only used to initialize some fields and that the hunks / changes will be overwritten.